lib/ovirt/template.rb

```diff
@@ -152,9 +152,7 @@ def build_clone_xml(options)
             end
           end
 
-          xml.os(:type => options[:os_type] || os_type) do
-            xml.boot(:dev => 'hd')
-          end
+          xml.os(:type => options[:os_type] || os_type)
```

Comment:
@bdunne Ok, but I don't see how this would be negatively affected by this change, since blank template should have the default boot order of `hd` so your skeletal clone should IMO still work.